pkg/pdapi/pdapi.go

```diff
@@ -315,7 +316,14 @@ func (c *pdClient) getStores(apiURL string) (*StoresInfo, error) {
 }
 
 func (c *pdClient) GetStores() (*StoresInfo, error) {
-	return c.getStores(fmt.Sprintf("%s/%s", c.url, storesPrefix))
+	storesInfo, err := c.getStores(fmt.Sprintf("%s/%s", c.url, storesPrefix))
+	if err != nil {
+		if strings.HasSuffix(err.Error(), tiKVNotBootstrapped+"\n") {
```

Comment:
Maybe use `pdapi.IsTiKVNotBootstrappedError`?

Reply:
`pdapi.IsTiKVNotBootstrappedError` is used to judge whether the type of error is `TiKVNotBootstrappedError`. Here is to return `TiKVNotBootstrappedError`.